examples/reminders/lib/feature_dashboard/views/dashboard_page.dart

```diff
@@ -81,14 +81,7 @@ class DashboardPage extends StatelessWidget implements AutoRouteWrapper {
         ),
       );
 
-  AppBar _buildAppBar(BuildContext context) => AppBar(
-        actions: [
-          IconButton(
-            icon: const Icon(Icons.refresh),
-            onPressed: context.read<DashboardBlocType>().events.fetchData,
-          ),
-        ],
-      );
+  AppBar _buildAppBar(BuildContext context) => AppBar();
```

Comment:
Do we really need a method that returns a single widget ?

Reply:
I have removed the method and moved the `AppBar()` in the `Scaffold`